esti/s3_gateway_test.go

```diff
@@ -76,34 +76,33 @@ func TestS3UploadAndDownload(t *testing.T) {
 				objects = make(chan Object, parallelism*2)
 			)
 
+			client := newMinioClient(t, sig.GetCredentials)
```

Comment:
Sorry!  Missed that the [MinIO client is thread-safe](https://github.com/minio/minio-go/issues/598#issuecomment-569457863), so linking this in here.